pandas/core/indexes/datetimes.py

```diff
@@ -1006,6 +1006,16 @@ def indexer_between_time(
 
         return mask.nonzero()[0]
 
+    def _convert_arr_indexer(self, keyarr):
```

Comment:
where exactly are these hit?

Reply:
these function call from "._convert_listlike_indexer" and get returned value from "common.asarray_tuple"